ts_src/address.ts

```diff
@@ -131,7 +124,12 @@ export function fromBlech32(address: string): Blech32Result {
   const pubkey = Buffer.from(result.blindingPublicKey, 'hex');
   const prg = Buffer.from(result.witness, 'hex');
   const data = Buffer.concat([
-    Buffer.from([result.witnessVersion, prg.length]),
+    Buffer.from([
+      result.witnessVersion
```

Comment:
better be explicit here ie. === 1

Reply:
Hum fromBlech32 should work with all witness version